.github/workflows/ci_special.yml

```diff
@@ -64,7 +64,7 @@ jobs:
     runs-on: ubuntu-22.04
     if: always()
     steps:
-      - uses: technote-space/workflow-conclusion-action@45ce8e0eb155657ab8ccf346ade734257fd196a5 # tag v3.0.3
+      - uses: ./.github/workflows/scripts//workflow-conclusion # no tag - locally sourced
```

Comment:
Same thought as on the other PR. I'd prefer these comments to be dropped as redundant.